internal/services/network/subnet_resource.go

```diff
@@ -201,6 +201,13 @@ func resourceSubnet() *pluginsdk.Resource {
 				},
 			},
 
+			"default_outbound_access_enabled": {
+				Type:     pluginsdk.TypeBool,
+				Computed: true,
+				Optional: true,
+				ForceNew: true,
```

Comment:
Since we're supporting Update below, this shouldn't be ForceNew?

```suggestion
```

Reply:
Changing outbound access requires deleting a re-creating the subnet. I've removed the update code.